projects/nae-example-app/src/app/doc/public-single-task-view/public-single-task-view.component.html

```diff
@@ -31,3 +31,12 @@
         <!-- TODO -->
     </mat-toolbar>
 </ng-template>
+
+<ng-template #noTaskSection>
+    <mat-icon color='accent'>check_box</mat-icon>
+    <span class='font-size-20'>{{ 'tasks.view.noTasksSatisfyingThisFilter' | translate }}</span>
+    <button mat-raised-button color="primary" class="public-button"
+            (click)="newCase();">
+        <span>Create new case</span>
```

Comment:
do we translate example-app texts? If yes, a translation is missing.

Reply:
We only translate some texts in example-app, not everything, but I added translation for this.

Reply:
that's an example....   I don't think it's necessary